### ISSUE TITLE
Clean up "Footers" Code

### ISSUE THREAD

**Issue by tailerchimchar** [tailerchimchar/therapywebsite#18]
So this idiot Derick made a footer, which looks great and all. However, he implemented a not so clean coding practice where he copy and pasted his Footer code into every single component. 

   {/* Footer Section */}
      <div className="footer mt-auto" style={{ backgroundColor: '#333', color: '#fff' }}>
        ......
      100 lines of code added to every single page 
    </div>

The task here is to modularize this code and add it as a component to the bottom of each page.

In other words, we don't want to copy and paste 50 lines of code each time. Either make a new component called footer.js or make a new file in components/helpers, or figure out a safe coding practice that we can use to make the code look cleaner.
